opensquirrel/ir.py

```diff
@@ -388,6 +406,7 @@ def named_gate(gate_generator: Callable[..., MatrixGate]) -> Callable[..., Matri
 def named_gate(gate_generator: Callable[..., ControlledGate]) -> Callable[..., ControlledGate]: ...
 
 
+# TODO: can we extract the duplicated code for the two wrapper functions below out to a single function?
```

Comment:
Please create an issue for this. We should limit the amount of `TODO`s in the code.

Reply:
Good idea, thanks!

Reply:
Issue created: https://github.com/QuTech-Delft/OpenSquirrel/issues/234